Updating Contrib (#181)
rust-unofficial/patterns: README.md

An open source book about design patterns and idioms in the Rust programming
language that you can read [here](https://rust-unofficial.github.io/patterns/).

## Contributing

You are missing content in this repository that can be helpful for others and you are eager to explain it?
Awesome! We are always happy about new contributions (e.g. elaboration or corrections on certain topics) to this project.

You can check the [Umbrella issue](https://github.com/rust-unofficial/patterns/issues/116) for all the patterns, anti-patterns, and idioms that could be added.

> We suggest reading our [Contribution guide](./CONTRIBUTING.md) to get more information on how contributing to this repository works.

Reviewer: ```suggestion
Be sure to read the [Contribution guide](./CONTRIBUTING.md), too!.
```
Sorry, I like brief and simple sentences 😂 Feel free to discard this if you don't like it

Author: I like the sentence before :-(
